ikarus/finiteelements/ferequirements.hh

```diff
@@ -112,6 +112,7 @@ namespace Ikarus {
     };
   }  // namespace Impl
 
+  // @todo FErequirements -> FERequirements

```

Comment:
can you remove this todo and write an issue instead, I would hope that we fix this todos directly, but I would propose the policy that no todo statements should be merged to master. This keeps it more clean